src/assets/sass/modules/Form/_form.scss

```diff
@@ -0,0 +1,158 @@
+.form-container {
+  position: absolute;
+  top: 0;
+  left: 0;
+  background: rgba($color: $black, $alpha: 0.7);
+  width: 100%;
+  height: 100vh;
+  z-index: 500;
+  overflow: auto;
+  display: flex;
+  align-items: center;
+  animation-name: animate;
+  animation-duration: 1s;
+}
+
+.form {
+  width: 502px;
+  margin: auto;
+  font: normal normal 12px/15px $font-primary;
+
+  @include e(header) {
+    font: normal bold 24px/29px $font-primary;
+  }
+
+  @include e(radio-field) {
+    margin-top: 18px;
+  }
+
+  @include e(radio-field-content) {
+    display: flex;
+    align-items: center;
+  }
+
+  .input {
```

Comment:
Keep this `input` outside of `form`. Keeping it inside will only create unnecessary nesting and make it more complex. you can create a separate `_input.scss` partial

Reply:
🆗